ast/policy_test.go

```diff
@@ -395,14 +395,16 @@ func TestExprEveryCopy(t *testing.T) {
 func TestRuleHeadEquals(t *testing.T) {
 	assertHeadsEqual(t, &Head{}, &Head{})
 
-	// Same name/key/value
+	// Same name/ref/key/value
 	assertHeadsEqual(t, &Head{Name: Var("p")}, &Head{Name: Var("p")})
+	assertHeadsEqual(t, &Head{Ref: Ref{VarTerm("p"), StringTerm("r")}}, &Head{Ref: Ref{VarTerm("p"), StringTerm("r")}}) // TODO: string for first section
```

Comment:
Is `StringTerm` something we still want to support for the first term?

Reply:
I've changed my mind 😅